api/system.cc

```diff
@@ -182,6 +183,17 @@ void set_system(http_context& ctx, routes& r) {
         apilog.info("Profile dumped to {}", profile_dest);
         return make_ready_future<json::json_return_type>(json::json_return_type(json::json_void()));
     }) ;
+
+    hs::get_highest_supported_sstable_version.set(r, [&ctx](const_req req) -> sstring {
+        auto supported_features = ctx.db.local().features().supported_feature_set();
```

Comment:
Although this will work, We can reuse `sstables_manager::get_highest_supported_format()`.
And get to it by looking up a table, e.g. `system.local`, and getting to its sstables_manager via `t.get_sstables_manager()`

Reply:
That's definitely a better approach.